README.rst

```diff
@@ -0,0 +1,23 @@
+django-oauth-plus
+=================
+
+.. image:: https://travis-ci.org/edx/django-oauth-plus.svg?branch=master
+    :target: https://travis-ci.org/edx/django-oauth-plus
+
+This is an edX-customized fork of *django-oauth-plus*. It is intended to be a short-term fork as we are planning on
+replacing this authentication backend soon.
```

Comment:
Nit: Should the ticket below be referenced here?:
https://openedx.atlassian.net/browse/PLAT-1660
(Perhaps this PR mention is enough...)